crowdin.yml

```diff
@@ -0,0 +1,17 @@
+project_id_env: CROWDIN_PROJECT_ID
+api_token_env: CROWDIN_PERSONAL_TOKEN
```

Comment:
@teolemon Since we moved the repo, the API key secret is not present any more. Could you possibly have a look at it, and check whether Crowdin understands the current config? Thank you! 🙏